openassessment/tests/test_data.py

```diff
@@ -363,13 +363,15 @@ class TestOraAggregateDataIntegration(TransactionCacheResetTest):
 
     def setUp(self):
         super(TestOraAggregateDataIntegration, self).setUp()
+        self.maxDiff = None
         # Create submissions and assessments
         self.submission = self._create_submission(STUDENT_ITEM)
         self.scorer_submission = self._create_submission(SCORER_ITEM)
         self.earned_points = 1
         self.possible_points = 2
         peer_api.get_submission_to_assess(self.scorer_submission['uuid'], 1)
         self.assessment = self._create_assessment(self.scorer_submission['uuid'])
+        self.assertEqual(self.assessment['parts'][0]['criterion']['label'], "criterion_1")
```

Comment:
It was a flaky test issue. A particular hashseed value caused the assessment to store out-of-order values, until I made [this fix](https://github.com/edx/edx-ora2/pull/1023/files/5d5921287a9c00b3935b86756cd960f64374b678#diff-c09750592365024b37e7601ab2b68bad). Now, this test ensures that `self.assessment['parts'][0]` matches up to the first criterion and appears first in the downloaded data.